teaser/examples/e9_building_data_import_from_excel.py

```diff
@@ -0,0 +1,629 @@
+volume (NetArea)
+- Floor area is only as big the respective net area of the heated room volume
+(NetArea)
+- Rooftop area is only as big the respective net area of the heated room
+volume (NetArea)
+- Rooftops are flat and not tilted, see "RooftopTilt"
+- Ceiling area is only as big the respective net area of the heated room
+volume (NetArea)
+- Ceiling, floor and inner walls are only respected by half their area,
+since they belong half to the respective
+and half to the adjacent zone
+- Orientations are clockwise in degree, 0° is directed north
+
+-respective construction types have to be added to the TypeBuildingElements.json
+-respective UsageTypes for Zones have to be added to the UseConditions.json
+-excel file format has to be as shown in the "ExcelBuildingData_Sample.xlsx"
+-yellowed columns are necessary input to teaser -> don´t change column
+header, keep value names consistent.
+-non yellowed columns may either not be used or be used for your zoning
+algorithm
+
+Information about the required excel format:
+#Documentation in progress!
+-Under the cell ‚Usage type‘  you will see some cells that are blank but have
+their row filled.
+It means the blank cell actually belongs to the Usage type above but in that
+specific row we filled the characteristics
+of the window/wall of a different orientation of the same exact room. That
+means every row is either a new room or a
+new orientation of that room. A room might have two outerwalls in two
+different orientation so for each outer wall,
+a an extra row defining the respective orientation is added
+-The entries in the excel sheet must be consistent for python being able to
+convert it.
+-If an inner wall is reaching inside a room but is not the limit of the room,
+it should be accounted with 2x the area
+"""
+
+import os
+from teaser.project import Project
+from teaser.logic.buildingobjects.building import Building
+from teaser.logic.buildingobjects.thermalzone import ThermalZone
+from teaser.logic.buildingobjects.useconditions import UseConditions
+from teaser.logic.buildingobjects.buildingphysics.outerwall import OuterWall
+from teaser.logic.buildingobjects.buildingphysics.floor import Floor
+from teaser.logic.buildingobjects.buildingphysics.rooftop import Rooftop
+from teaser.logic.buildingobjects.buildingphysics.groundfloor import GroundFloor
+from teaser.logic.buildingobjects.buildingphysics.ceiling import Ceiling
+from teaser.logic.buildingobjects.buildingphysics.window import Window
+from teaser.logic.buildingobjects.buildingphysics.innerwall import InnerWall
+import pandas as pd
+import numpy as np
+import warnings
+import shutil
+
+
+def import_data(path=None, sheet_names=None):
+    """
+    Import data from the building data excel file and perform some
+    preprocessing for nan and empty cells.
+    If several sheets are imported, the data is concatenated to one dataframe
+
+    :param path: path to the excel file that should be imported
```

Comment:
this is not the documentatin style guide we use in TEASER :( sorry, can you adjust that?

Reply:
Thanks for noticing, the style should now match.